notebooks/generative-ai/question-answering.ipynb

```diff
@@ -172,12 +203,17 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 58,
       "metadata": {},
       "outputs": [],
       "source": [
         "documents = vector_store.from_documents(\n",
-        "    docs, embeddings, es_cloud_id=CLOUD_ID, es_user=CLOUD_USERNAME, es_password=CLOUD_PASSWORD, index_name=\"workplace_index\"\n",
+        "    docs, \n",
+        "    embeddings,  \n",
+        "    index_name=\"workplace_index\",\n",
+        "    es_cloud_id=CLOUD_ID,\n",
+        "    es_user=CLOUD_USERNAME,\n",
+        "    es_password=CLOUD_PASSWORD\n",
```

Comment:
This is only appropriate for a relatively short list of documents, right? You would never take this approach for indexing a large collection of docs. Should we show how to work with a large dataset, either instead of in addition to this short dataset example? Or at least mention that this is not a scalable solution?

Reply:
Its also fine for ingesting large amount of docs, which developers use for this. Behind the scenes it uses bulk helper which chunks requests.

Reply:
Okay, so then I don't understand how this works. What does the `docs` local variable holds? It seems it has the entire database of chunked documents. My thinking was that this is only viable when your entire database fits in memory, which I think is probably not the case for a lot of people.

Reply:
discussed offline.